src/common/config/src/wal/kafka.rs

```diff
@@ -60,7 +60,8 @@ impl Default for KafkaConfig {
         Self {
             broker_endpoints: vec!["127.0.0.1:9092".to_string()],
             compression: RsKafkaCompression::NoCompression,
-            max_batch_size: ReadableSize::mb(4),
+            // Warning: Kafka has a default limit of 1MB per message in a topic.
```

Comment:
```suggestion

```

Reply:
Let's keep this comment to prevent someone setting a too-large default `max_batch_size` in production or in testing.